pyro/contrib/timeseries/gp.py

```diff
@@ -366,7 +367,7 @@ def _trans_matrix_distribution_stat_covar(self, dts):
 
     def _get_dist(self):
         """
-        Get the `GaussianHMM` distribution that corresponds to a `DependentMaternGP`
+        Get the :class:`GaussianHMM` distribution that corresponds to a :class:`DependentMaternGP`
```

Comment:
ditto, fully qualify the GaussianHMM. (I believe the DependentMaternGP need not be fully qualify because it is in this module)